.ci/setenvconfig

```diff
@@ -310,6 +310,23 @@ ENV
   fi
 ;;
 
+######################################
+  dev/build)
+######################################
+
+write_env <<ENV
+REGISTRY = docker.elastic.co
+REPOSITORY = eck-snapshots
+IMG_SUFFIX = -dev-ci
+SNAPSHOT_TEST_TARGET = yes
```

Comment:
Any reason to override what has been "guessed" at line 204 ?

Reply:
Yes. L204 is for a different environment. Basically there are two environments where it is relevant whether we are building or testing against snapshot builds:
1. when building the operator image. Here we know we want to build a dev version that can deal with snapshot stack versions so we can hard-code `SNAPSHOT_TEST_TARGET` to pseudo-boolean `yes`. This is what this line is about
2. when running the e2e tests, here I wanted to use the same environment config for both variants so we do some auto-detection based on the stack version. That's line 204

Reply:
But I realise now that `SNAPSHOT_TEST_TARGET` is a bad name for case 1.